src/Serilog.AspNetCore/AspNetCore/SerilogLoggerFactory.cs

```diff
@@ -18,26 +18,49 @@
 
 namespace Serilog.AspNetCore
 {
-    class SerilogLoggerFactory : ILoggerFactory
+	/// <summary>
+	/// Implements Microsoft's ILoggerFactory so that we can inject Serilog Logger.
+	/// </summary>
+	/// <seealso cref="Microsoft.Extensions.Logging.ILoggerFactory" />
+	public class SerilogLoggerFactory : ILoggerFactory
     {
         readonly SerilogLoggerProvider _provider;
 
-        public SerilogLoggerFactory(Serilog.ILogger logger = null, bool dispose = false)
+		/// <summary>
+		/// Initializes a new instance of the <see cref="SerilogLoggerFactory"/> class.
+		/// </summary>
+		/// <param name="logger">The logger.</param>
+		/// <param name="dispose">if set to <c>true</c> [dispose].</param>
```

Comment:
Could use the same documentation here as in https://github.com/serilog/serilog-aspnetcore/blob/dev/src/Serilog.AspNetCore/SerilogWebHostBuilderExtensions.cs#L33